client/src/util/errors.ts

```diff
@@ -1,4 +1,6 @@
 export const errors = {
+  noFetchImplementation: 'No fetch implementation provided',
+  invalidApiProvider: 'Invalid API provider',
   falsyFetchImplementation: `The \`fetch\` option passed to the Xata client is resolving to a falsy value and may not be correctly imported.
```

Comment:
Not sure about having an ``errors`` dictionary. Code looks cleaner but makes debugging harder. I saw a comment on a previous PR, not sure what was the outcome.

Reply:
No strong opinions. I prefer to have the message where it is used, because normally you have different errors for each case and that way you can quickly find where a message is being used.

Reply:
Since we are not consistent, removing them and inlining inside the error constructors